src/renderer/components/atoms/FailoverImg.vue

```diff
@@ -46,11 +48,14 @@ export default {
 
 <style lang="scss" scoped>
 img {
-  min-width: 20px;
-  min-height: 20px;
-  background-image: url("../../assets/images/loading-spinner.svg");
-  background-position: center center;
-  background-repeat: no-repeat;
-  background-size: contain;
+  min-width: 1em;
+  min-height: 1em;
+
+  &.loading {
+    background-image: url("../../images/loading-spinner.svg");
```

Comment:
Why change image path? Does this image exist in this path?

Reply:
Apologies, I rebased this blindly upon f4cb57ff08a9610deedb387349edd2165066a8e5 without checking. I've fixed it.